.github/workflows/pr-check.yml

```diff
@@ -0,0 +1,30 @@
+name: PR Check
+
+on:
+  pull_request:
+    branches: 
+      - main
+
+jobs:
+  build:
+    name: Build playground
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [18.x]
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - name: NPM Install
+        run: npm install
+
+      - name: NPM Build
+        run: SERVER_ROOT=https://playground.accordproject.org && NODE_OPTIONS=--max_old_space_size=8192 npm run build
```

Comment:
Should this run tests too? If we don't have tests, let's add an empty npm script so that they will start running once we add them

Reply:
Working on that next, add rtl and playwright for testing, I'll add add the test script here too, including a coverage check. #60